cosinnus_event/models.py

```diff
@@ -705,10 +706,15 @@ class ConferenceEvent(Event):
         help_text='If an event is a break, no rooms will be created for it, and it will be displayed differently',
         default=False)
 
+    # Checkbox for visible / hidden mode of the particular event
+    is_visible = models.BooleanField(_('Event is visible'), 
+        help_text='Provides an option to choose if the particular event should be shown on microsite or not',
+        default=True)
+
     # Checkbox for public / private mode of the event's description
-    is_confidential = models.BooleanField(_('Event is confidential'), 
+    is_public = models.BooleanField(_('Event is public'), 
```

Comment:
Same here, please rename the field so it is clearer what it does. Something like `is_description_visible_on_microsite` or similar. `is_public` is confusing as it implies that the event itself would be made public with this switch.